packages/security/src/deviceTrust/SecurityCheckType.ts

```diff
@@ -1,7 +1,9 @@
 import { RootedCheck } from "./checks";
+import { DebuggerCheck } from "./checks";
 
 export class SecurityCheckType {
   public static readonly IsRooted = new RootedCheck();
+  public static readonly IsDebugger = new DebuggerCheck();
```

Comment:
Sorry for the fastidiousness but isn't **debugger** a bit misleading? What about `isDebugMode` or simply `isDebug`. Debugger somewhat suggest that this detects if the debugger is being used while it is only detecting wether the app is on debug or release mode.

Reply:
`isDebuggerConnected` might be more accurate if we are following what we used in the Android SDK - https://developer.android.com/reference/android/os/Debug#isdebuggerconnected

Reply:
But what are we detecting exactly? The attached debugger or the build-config called "debug"? (aka development mode)

Reply:
@josemigallas A connected java debugger. The build config debug flag is under `ApplicationInfo.FLAG_DEBUGGABLE`.

Reply:
But looks like the plugin we are using only checks for that flag - https://github.com/mattlewis92/cordova-plugin-is-debug/blob/master/src/android/IsDebug.java#L34 

On the other hand, the Android/iOS templates/SDK's actually check if a debugger is attached.

Reply:
Those are two completely different things. Debug means that you can use for example self signed certs etc. IMHO Checking if debugger is attached has not much value.

Reply:
@TommyJ1994 Do you think the current plugin we're using here does what we need? Or should we be looking for a new plugin/create our own.

Reply:
@aidenkeating Yeah - This one was actually created for detecting that case - checking if you do not send debug version somewhere where you should not.

Reply:
Well we are trying to defend against the *action* of a malicious user debugging as per OWASP/NowSecure guidelines. We could do both, don't care or just make a note, but was just giving a heads up that the cordova implementation is different to android/iOS currently.

Reply:
Actually maybe we can just open conversation on the list as it will be quite easy to get both done, but we need to get consensus if we need both or at least be consistent in naming and name this one differently so it will appear in different category in grafana metrics.